packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.module.scss

```diff
@@ -33,227 +45,359 @@
-        @include bpk-heading-1;
+      > * {
+        margin: initial;
       }
+    }
 
-      &:not(:last-child) {
-        margin-bottom: bpk-spacing-md();
+    @include bpk-breakpoint-small-mobile {
+      margin: initial;
+      padding: bpk-spacing-lg();
+      justify-content: space-between;
+      align-items: initial;
+
+      @include fill-parent;
+
+      &--inverted {
+        flex-direction: column-reverse;
+      }
+
+      > * {
+        margin: initial;
       }
     }
   }
 
-  @include breakpoint-portrait-large {
-    &__sponsor-label {
+  &__tagline {
+    @include bpk-breakpoint-above-tablet {
       margin-bottom: bpk-spacing-md();
 
-      @include bpk-heading-4;
+      @include bpk-label-1;
     }
 
-    &__tagline {
+    @include bpk-breakpoint-tablet {
       margin-bottom: bpk-spacing-md();
 
-      @include bpk-subheading;
+      @include bpk-label-1;
     }
 
-    &__headline {
+    @include bpk-breakpoint-small-tablet {
       margin-bottom: bpk-spacing-md();
 
-      @include bpk-hero-4;
-    }
-
-    &__subheading {
       @include bpk-subheading;
     }
-  }
 
-  @include breakpoint-portrait-medium {
-    &__sponsor-label {
+    @include bpk-breakpoint-mobile {
       margin-bottom: bpk-spacing-md();
 
-      @include bpk-heading-3;
+      @include bpk-subheading;
     }
 
-    &__tagline {
+    @include bpk-breakpoint-small-mobile {
       margin-bottom: bpk-spacing-md();
 
-      @include bpk-subheading;
+      @include bpk-label-1;
     }
+  }
 
-    &__headline {
-      margin-bottom: bpk-spacing-md();
-
+  &__headline {
+    @include bpk-breakpoint-above-tablet {
       @include bpk-hero-5;
+
+      &:not(:last-child) {
+        margin-bottom: bpk-spacing-md();
+      }
     }
 
-    &__subheading {
-      @include bpk-body-longform;
+    @include bpk-breakpoint-tablet {
+      @include bpk-heading-1;
+
+      &:not(:last-child) {
+        margin-bottom: bpk-spacing-md();
+      }
     }
-  }
 
-  @include breakpoint-portrait-small {
-    &__sponsor-label {
-      margin-bottom: bpk-spacing-sm();
+    @include bpk-breakpoint-small-tablet {
+      margin-bottom: bpk-spacing-md();
 
-      @include bpk-label-1;
+      @include bpk-heading-1;
     }
 
-    &__tagline {
+    @include bpk-breakpoint-mobile {
       margin-bottom: bpk-spacing-md();
 
-      @include bpk-label-1;
+      @include bpk-hero-5;
     }
 
-    &__headline {
+    @include bpk-breakpoint-small-mobile {
       margin-bottom: bpk-spacing-md();
 
       @include bpk-heading-2;
     }
+  }
+
+  &__subheading {
+    @include bpk-breakpoint-above-tablet {
+      @include bpk-body-default;
+    }
+
+    @include bpk-breakpoint-tablet {
+      @include bpk-body-default;
+    }
 
-    &__subheading {
+    @include bpk-breakpoint-small-tablet {
+      @include bpk-subheading;
+    }
+
+    @include bpk-breakpoint-mobile {
+      @include bpk-body-longform;
+    }
+
+    @include bpk-breakpoint-small-mobile {
       @include bpk-body-default;
     }
```

Comment:
Here you can simplify this and only need to override when its at a breakpoint, so:

```
@include bpk-body-default
@include bpk-breakpoint-small-tablet {
  @include bpk-subheading;
}

@include bpk-breakpoint-mobile {
  @include bpk-body-longform;
}

```

Reply:
Sorted. 🙂